### DISCUSSION TITLE
Docs on .env options?

### DISCUSSION THREAD

**Discussion by brywhi** [OpnForm/OpnForm#523] [Q&A]
I'm a bit confused about environment variables. Specifically, I'm looking to enable smtp so I can invite other users, etc. but I can't figure out how to do that. Are there docs about the .env and client/.env files? Thanks!

**Comment by JhumanJ** [comment]
Hi there, you can find more info about config for email in the Laravel documentation: https://laravel.com/docs/11.x/mail.
We will soon work on creating our own documentations to simplify this whole process! Essentially you just need to add smtp config to your .env files
